itou/utils/immersion_facile.py

```diff
@@ -0,0 +1,29 @@
+from urllib.parse import quote, urlencode
+
+from itou.utils.constants import IMMERSION_FACILE_SITE_URL
+
+
+def immersion_facile_search_url(user):
```

Comment:
Je dit ça que tu y penses les prochaines fois mais mais la fonction étant dans un module `immersion_facile` c'est redondant (en plus d'être long) de la faire commencer aussi par "immersion_facile".
Vu qu'on a tendance à importer directement les symboles ça simplifie l'import mais si on importe le module on se retrouve avec `immersion_facile.immersion_facile_search_url()`, ce qui est assez inélégant.

Reply:
C'est vrai mais le nom `immersion_facile_search_url` est plus clair (que `search_url`), et j'ai choisis une module `immersion_facile` pour faciliter la possibilité qu'un jour on va déménager ce code

À la recherche d'un compromis j'ai changé le nom à `immersion_search_url`